plugins/fclrave/fclspace.h

```diff
@@ -82,6 +84,27 @@ class FCLSpace : public boost::enable_shared_from_this<FCLSpace>
     class KinBodyInfo : public boost::enable_shared_from_this<KinBodyInfo>, public OpenRAVE::UserData
     {
 public:
+        class GeometryInfo
+        {
+public:
+            GeometryInfo() : bFromKinBodyGeometry(false) {
+            }
+
+            GeometryInfo(KinBody::GeometryPtr pgeom) : _pgeom(pgeom), bFromKinBodyGeometry(true) {
+            }
+
+            virtual ~GeometryInfo() {
+            }
+
+            KinBody::GeometryPtr GetGeometry() {
```

Comment:
let's inline this, and also ``LinkInfo::GetLink``

Reply:
I slipped my mind. I made them inline now.